### ISSUE TITLE
Missing unicode_segmentation::GraphemeIndices equivalent struct

### ISSUE THREAD

**Issue by gwenn** [dahosek/finl_unicode#16]
`unicode_segmentation::Graphemes` can be replaced by `finl_unicode::grapheme_clusters::Graphemes`.
But there is no replacement for `unicode_segmentation::GraphemeIndices`.
Would you be interested by providing one ?

**Comment by dahosek** [comment]
I can put it on the roadmap. Do you have a use case for this that I can use for benchmarking/testing?

**Comment by gwenn** [comment]
Not really, but [termwiz](https://github.com/wez/wezterm/blob/main/termwiz/Cargo.toml) should be able to remove `unicode_segmentation` (and keep only `finl_unicode`).
And [rustyline](https://github.com/kkawakam/rustyline/blob/master/Cargo.toml#L36) could replace `unicode_segmentation` by `finl_unicode`.
